src/main/java/uk/gov/hmcts/reform/hmc/exceptions/ValidationError.java

```diff
@@ -159,7 +159,6 @@ private ValidationError() {
     public static final String HEARING_ACTUALS_ID_NOT_FOUND = "001 No such id: %s";
     public static final String HEARING_ACTUALS_INVALID_STATUS = "002 invalid status";
     public static final String HEARING_ACTUALS_MISSING_HEARING_OUTCOME = "005 missing hearing outcome";
-    public static final String HEARING_ACTUALS_MISSING_HEARING_DAY = "003 missing hearing day actuals";
     public static final String HEARING_ACTUALS_UN_EXPRECTED = "004 unexpected hearing day actuals";
```

Comment:
typo - should it be `UNEXPECTED` ?

Again, it's not your changes - might be worth raising some of these as technical debt/nice to have

Reply:
fixed type in 5b54fe65756a70e290c182eb7b7e1216c99e4f30